py/server/deephaven/table_listener.py

```diff
@@ -546,15 +555,15 @@ def stop(self) -> None:
         if not self.started:
             return
 
-        with update_graph.auto_locking_ctx(self.listener_recorders[0].table.update_graph):
+        with update_graph.auto_locking_ctx(self.tables[0]):
```

Comment:
Having the lock *does* make sense for the merged case. Otherwise, you may process inconsistent updates.

Reply:
A comment should be added on this detail